{{ cookiecutter.project_shortname }}/.travis.yml

```diff
@@ -24,12 +24,12 @@ python:
 before_install:
   - "travis_retry pip install --upgrade pip setuptools py"
   - "travis_retry pip install twine wheel coveralls requirements-builder"
-  - "requirements-builder --level=min setup.py > .travis-lowest-requirements.txt"
-  - "requirements-builder --level=pypi setup.py > .travis-release-requirements.txt"
-  - "requirements-builder --level=dev --req requirements-devel.txt setup.py > .travis-devel-requirements.txt"
+  - "requirements-builder --level=min setup.py > built-requirements-lowest.txt"
```

Comment:
Why not to stick to the names used in [requirements-builder's own documentation](http://pythonhosted.org/requirements-builder/#travisci)? People coming from (or coming to) other Python projects may feel at home...